lms/djangoapps/django_comment_client/tests/test_utils.py

```diff
@@ -118,7 +120,7 @@ class CoursewareContextTestCase(ModuleStoreTestCase):
     comment client service integration
     """
     def setUp(self):
-        super(CoursewareContextTestCase, self).setUp(create_user=True)
+        super(CoursewareContextTestCase, self).setUp()
```

Comment:
If I'm reading this right, I think that CREATE_USER = False has only been set for the AccessUtilsTestCase testcase class and needs to be set at the class level for the rest of the testcase classes in this file.